### PR TITLE
create the landing page

### PR DESCRIPTION
**Pull request by junaidshaikh-js**
Things added 

- navigation bar with sidebar on mobile screen
- hero section 
- footer with social links

### CODE REVIEW

File: css/base.css [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+}
+
+body {
+  background-color: var(--background-color);
+  font-family: var(--seconary-font-family);
+  line-height: 1.5;
+  overflow-x: hidden;
+}
+
+.visually-hidden {
```
Comment (ritik-patel05) [inline css/base.css]: can move this to utility CSS. I guess we should restrict ourselves to resetting browser style to all elements.

File: css/base.css [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  height: 1px;
+  left: 0;
+  overflow: hidden;
+  position: absolute;
+  top: -500px;
+  width: 1px;
+}
+
+[class^="f"] {
```
Comment (ritik-patel05) [inline css/base.css]: same here, can be moved to utility

File: css/base.css [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+  --text-black-hover: rgba(0, 0, 0, 0.85);
+  --text-hover-background: rgba(121, 82, 179, 0.1);
+  --logo-font-family: "Mochiy Pop P One", sans-serif;
+  --primary-font-family: "Montserrat", sans-serif;
+  --seconary-font-family: "Roboto Mono", monospace;
+  --font-size-sm: 1.2rem;
+  --font-size-lg: 1.4rem;
+}
+
+html {
```
Comment (ritik-patel05) [inline css/base.css]: move to reset

File: css/base.css [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+  scroll-behavior: smooth;
+}
+
+body {
```
Comment (ritik-patel05) [inline css/base.css]: move to reset

File: css/base.css (hunk 1 above)
Comment (ritik-patel05) [reply css/base.css]: can keep here

File: css/base.css (hunk 2 above)
Comment (ritik-patel05) [reply css/base.css]: can keep here

File: css/base.css [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+  top: -500px;
+  width: 1px;
+}
+
+[class^="f"] {
+  font-size: 30px;
+  color: inherit;
+}
+
+h1,
```
Comment (ritik-patel05) [inline css/base.css]: move to reset

File: css/base.css [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+h2,
+h3,
+h4,
+h5,
+h6 {
+  margin: 3.5rem 0 2rem 0;
+  font-family: var(--primary-font-family);
+}
+
+p {
```
Comment (ritik-patel05) [inline css/base.css]: move to reset

File: css/utility.css [hunk 7]
```diff
@@ -0,0 +1,135 @@
+/* grid */
+.grid-two-column-layout {
+  display: grid;
+  grid-template-columns: 1fr;
+}
+
+.g-2rem {
```
Comment (ritik-patel05) [inline css/utility.css]: can remove "rem" from g-, p-, space, plr, ptb, mt

File: css/utility.css [hunk 8]
```diff
@@ -0,0 +1,135 @@
+/* grid */
+.grid-two-column-layout {
```
Comment (ritik-patel05) [inline css/utility.css]: .grid-2-col

File: css/utility.css [hunk 9]
```diff
@@ -0,0 +1,135 @@
...
+
+.text-center {
+  text-align: center;
+}
+
+.bold {
+  font-weight: bold;
+}
+
+/* components */
```
Comment (ritik-patel05) [inline css/utility.css]: elements

File: css/utility.css [hunk 10]
```diff
@@ -0,0 +1,135 @@
...
+  max-width: 1200px;
+  margin: 0 auto;
+}
+
+.max-1000-center {
+  max-width: 1000px;
+  margin: 0 auto;
+}
+
+h1 {
```
Comment (ritik-patel05) [inline css/utility.css]: move to reset.css

File: css/utility.css [hunk 11]
```diff
@@ -0,0 +1,135 @@
...
+  font-size: 2rem;
+}
+
+p {
```
Comment (ritik-patel05) [inline css/utility.css]: move to reset.css

File: css/utility.css [hunk 12]
```diff
@@ -0,0 +1,135 @@
...
+  font-size: 1.25rem;
+}
+
+/* style */
+.background-secondary {
```
Comment (ritik-patel05) [inline css/utility.css]: bg-secondary or bg-sec

File: css/utility.css [hunk 13]
```diff
@@ -0,0 +1,135 @@
...
+  background-color: var(--secondary-color);
+}
+
+@media screen and (min-width: 600px) {
```
Comment (ritik-patel05) [inline css/utility.css]: media queries on semantic elements to reset.css

File: css/utility.css [hunk 14]
```diff
@@ -0,0 +1,135 @@
...
+    padding: 0 3rem;
+  }
+}
+
+@media screen and (min-width: 1000px) {
+  .grid-two-column-layout {
+    grid-template-columns: 50% 50%;
+  }
+
+  h1 {
```
Comment (ritik-patel05) [inline css/utility.css]: here also

**Review by ritik-patel05 — COMMENTED** [review]
Can have separate folders of component library and documentation website

File: css/base.css (hunk 1 above)
Comment (junaidshaikh-js) [reply css/base.css]: Done

**Review by ritik-patel05 — APPROVED** [review]
LGTM

**Pull request merged**